src/containers/BooksList.js

```diff
@@ -1,35 +1,35 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
 import Book from '../components/Book';
+import { removeBook } from '../actions/index';
 
-const TABLE = styled.table`
-  width: 600px
-  height: 400px
-`;
-
-const BooksList = props => {
-  const { state } = props;
-  return (
-    <TABLE className="table-one">
-      <thead>
-        <tr>
-          <th>Book ID</th>
-          <th>Title</th>
-          <th>Category</th>
-        </tr>
-      </thead>
-      <tbody>
-        {state.map(book => <Book key={book.id} book={book} />)}
-      </tbody>
-    </TABLE>
-  );
-};
+const BooksList = ({ state, removeBook }) => (
```

Comment:
* [ ] it implements `handleBookRemove` [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)
- [ ] Instead of `state`, we could use a more descriptive name, like book. `state` is general in `React`, hence, it's going to be confusing.

```suggestion
const BooksList = ({ book, removeBook }) => {(
const handleRemoveBook = () => removeBook(book)
```